articles/vmware/vmw-how-access-ubuntu-update-servers.md

```diff
@@ -35,13 +35,17 @@ UKCloud's Elevated OFFICIAL platform doesn't natively connect to the internet, a
 
 We provide Ubuntu repositories for the 12.04 and 14.04 LTS versions of Ubuntu on our Elevated OFFICIAL platform, so that customers with Ubuntu VMs can get software updates. To do this, issue the appropriate command for your release of Ubuntu to download a repository list file to your apt sources directory.
```

Comment:
Does 12.04 and 14.04 here need changing to 16.04 and 18.04?

Reply:
Apologies, yes, it does. please remove references to 12.04 and 14.04